src/xg/test/testxg.cpp

```diff
@@ -154,3 +155,8 @@ int test(std::ostream &outStream)
 		return 1;
 	}
 }
+
+int main()
+{
+	return test(std::cout);
+}
```

Comment:
Sorry, I have been on vacation. I will try to set up an andriod build environment this weekend too see if I can work things out.